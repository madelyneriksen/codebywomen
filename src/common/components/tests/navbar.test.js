import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../navbar';


describe("Navbar", () => {
  it("Matches Snapshot", () => {
    const data = {
      site: {
        siteMetadata: {
          title: "CodeByWomen",
        },
      },
    };
    const tree = renderer.create(<Navbar data={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
