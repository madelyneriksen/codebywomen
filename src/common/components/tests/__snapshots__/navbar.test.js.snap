// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Matches Snapshot 1`] = `
<div
  class="bg-white h3 flex items-center pa2"
>
  <span
    class="f3 sans-serif"
  >
    CodeByWomen
  </span>
</div>
`;
